Tests/test_PDB.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2021 by Simon Duerr.  All rights reserved.
```

Comment:
I would move these tests to the respective parsers test files.

Reply:
But where would the tests for the fmt string and so on go? They are not related to the parsers directly right?  
To me it is logical to have the tests that test the read function itself in `test_PDB` and to have the tests that check the actual parsers e.g in `test_PDB_PDBParser` etc.

Reply:
They are somewhat directly related to the parsers, but I see your point. Let's keep them here.